scripts/install/opt/seagate/cortx/motr/bin/motr_setup

```diff
@@ -122,10 +122,8 @@ class PostInstallCmd(Cmd):
         if not lvm_exist(self):
             config_lvm(self)
         else:
-            self.logger.warning("lvm already exists. "
-                             "Skipping lvm creation.\n")
-            self.logger.warning("Updating bgsize.\n")
-            update_bgsize(self)
+            self.logger.error("There are LVM configurations present on the system\n")
+            sys.exit()
```

Comment:
Post install can be called again, so in such cases LVM's created in previous call are valid.

Just make sure that if valid LVM's are present proceed ahead, otherwise exit with error message that "Invalid/unclean LVM's are present and need to wipe them and re-run the step."